src/main/java/pl/orki/hackathon/webapp/user/boundary/UserConverter.java

```diff
@@ -1,18 +1,40 @@
 package pl.orki.hackathon.webapp.user.boundary;
 
+import org.jetbrains.annotations.NotNull;
+import pl.orki.hackathon.webapp.band.boundary.dto.BandResponseDTO;
 import pl.orki.hackathon.webapp.user.entity.User;
+import pl.orki.hackathon.webapp.venue.boundary.dto.VenueResponseDTO;
 
 public class UserConverter {
```

Comment:
Could you convert this class to a `@Component` without static methods? We should avoid using static methods, this was a temporary solution during hackathon ;)